File: cmd/feature-experimentation/analyze/code-samples/openfeature/java/sample-OF.java
public void example() {

    // flags defined in memory
    Map<String, Flag<?>> myFlags = new HashMap<>();
    myFlags.put("v2_enabled", Flag.builder()
            .variant("on", true)
            .variant("off", false)
            .defaultVariant("on")
            .build());

    // configure a provider
    OpenFeatureAPI api = OpenFeatureAPI.getInstance();
    api.setProviderAndWait(new InMemoryProvider(myFlags));

    // create a client
    Client client = api.getClient();

    // get a bool flag value
    boolean flagValue = client.getBooleanValue("of_java_v2_enabled", false);
}

File: cmd/feature-experimentation/analyze/code-samples/openfeature/php/sample-OF.php
use OpenFeature\OpenFeatureAPI;
use OpenFeature\Providers\Flagd\FlagdProvider;

function example()
{
$api = OpenFeatureAPI::getInstance();

// configure a provider
$api->setProvider(new FlagdProvider());

// create a client
$client = $api->getClient();

// get a bool flag value
$client->getBooleanValue('of_php_v2_enabled', false);
}